modules/get-started/pages/quickstarts/helm-chart.adoc

```diff
@@ -25,13 +25,37 @@ Common use cases for Redpanda Connect include:
 
 To deploy Redpanda Connect on Kubernetes, use the official Helm chart.
 
+You can optionally provide an enterprise license to enable enterprise connectors. If you have a license, follow these steps before installing:
+
+. Optional: Create a Kubernetes Secret with your license key:
++
+[source,bash]
+----
+kubectl create secret generic redpanda-connect-license \
+  --from-literal=license=<license-key> \
+  --namespace <namespace>
+----
+
+. Optional: Reference the license Secret in your `values.yaml` or as a Helm value:
```

Comment:
I think it's a bit confusing that these two steps are seperately optional, really the block is optional, but if you add a secret, adding the env var is not really optional.